lib/seahorn/BvOpSem2RawMemMgr.cc

```diff
@@ -816,6 +816,12 @@ RawMemManagerCore::getGlobalVariableInitValue(const GlobalVariable &gv) {
 bool RawMemManagerCore::isPtrTyVal(Expr e) {
   return (!e || !strct::isStructVal(e));
 }
+RawMemManagerCore::MemValTy
+RawMemManagerCore::setMemory(unsigned int val) const {
```

Comment:
the name `setMemory` is confusing because it is interpreted either as a setter (which it is not), or something that changes memory region (which it is not). The implementation seems to be a wrapper over `FilledMemory` that returns initialized memory region. Maybe rename all into `InitMem` or something similar to reflect the intention of the function.